### PR TITLE
[hotfix] update setup guide based on kristin's feedback

### PR DESCRIPTION
**Pull request by suzinyou**
Reviewer: @kristini-ho @amiraliemami 
Estimate: 10min

---

## Ticket

Fixes: none

### Goal

Reflecting @kristini-ho 's recommendations & fixing some formatting to make it easy to read

### Changes

1. Prerequisites for each setup method
2. Move pros and cons to bottom because the star of the show should be how to set up?
3. Some other fixes

## How has this been tested?

Run mkdocs at root to see how it reads

```
mkdocs serve -a "localhost:8080"
```

## Checklist

Fill with `x` for completed. Delete any lines that are not relevant

- [x] My code follows the style guidelines of this project
- [x] I have reviewed my own code to ensure good quality
- [x] I have tested the functionality of my code to ensure it works as intended
- [x] I have resolved merge conflicts
- [ ] I have updated the automated tests (if applicable)
- [ ] I have updated the requirements (if applicable)
- [ ] I have updated the README file (if applicable)
- [x] I have updated affected documentation (if applicable)

### CODE REVIEW

File: docs/develop/contributing.md [hunk 1]
```diff
@@ -60,22 +64,18 @@ You can automatically create a ready-to-go `aaq-core` conda environment with:
 
         git checkout -b feature
 
-2. Make your changes and then run `pre-commits` if you have not set it up to run automatically.
-
-        pre-commit run --all  # 👈 to run it manually
```
Comment (suzinyou) [inline docs/develop/contributing.md:65]: This should not be manual! I ran this, saw that we've accumulated lots of uncleaned files (extra whitespace, no new line, etc.). Not a huge deal but keeps commits & PRs clean

File: docs/develop/setup.md [hunk 2]
```diff
@@ -115,8 +115,20 @@ This will install the required packages required for the admin app and start the
 
 The admin app will now be accessible on [http://localhost:3000/](http://localhost:3000/)
 
-## Setting up docs
+## Set up docs
+
+Install [mkdocs](https://www.mkdocs.org/user-guide/installation/).
```
Comment (amiraliemami) [inline docs/develop/setup.md:120]: And also activate the environment in which you installed it.

**Review by amiraliemami — APPROVED** [review]
Thanks! I wonder if we should also add the setting up your environment bit to setup instead of contributing...

File: docs/develop/setup.md (hunk 2 above)
Comment (suzinyou) [reply docs/develop/setup.md]: Ah right. Added!

**Comment by suzinyou** [comment]
> Thanks! I wonder if we should also add the setting up your environment bit to setup instead of contributing...

True.. for now let's hope linking back to contributing.md is good enough. Made fixes based on your suggestions! merging

**Pull request merged**